index.html

```diff
@@ -7,14 +7,20 @@
 	<title>Portfolio</title>
 	<script src="https://kit.fontawesome.com/b8019b4db4.js" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="styles.css">
+
 	
 </head>
 
 <body>
 	<header class="container">
 		<div id="logo">Lassana</div>
-		<nav>
+		<nav class="nae toggle-menu">
 			<button id="hide" type="button"><i class="fas fa-bars"></i></button>
+			<ul class="mobile-menu" id="anchors">
+				<li class="pad"><a href="#works">Portfolio</a></li>
+				<li  class="pad"><a href="#about-myself">About</a></li>
+				<li  class="pad"><a href="#contact">Contact</a></li>
+			  </ul>
```

Comment:
Let's try to stick to the Figma design with the `header`. Currently, it's taking more space than we want. Here is your design compared with the Figma design.

| Your Design | Figma Design |
|-----------|--------------|
| <img width="453" alt="image" src="https://user-images.githubusercontent.com/34578631/162193590-eefcd4d8-dd7d-43aa-ab8a-59cc345165ab.png"> | <img width="404" alt="image" src="https://user-images.githubusercontent.com/34578631/162193672-d0c5cbcc-0c45-47d6-9a5b-0c07b34e1dc7.png"> |

Kindly, fix this issue.